<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>657</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe Devanagari</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>线性插值及矢量线性插值关键帧动画</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1001</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_grain">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>插值帧个数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_grain"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_speed">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>刷新频率（ms/次）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_speed"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_number">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>关键点个数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="number2">
           <property name="font">
            <font>
             <family>Adobe Devanagari</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>    0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>生成初始帧图形</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="end">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>生成终点帧图形</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="line">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>线性插值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="vector1">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>矢量线性插值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="vector2">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>矢量线性插值（顺时针）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="vector3">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>矢量线性插值（逆时针）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="change">
         <property name="text">
          <string>改变画笔颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>清空窗口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
